### ISSUE TITLE
Android [don't save when i change aplication]

### ISSUE THREAD

**Issue by USERCALL** [nspire-emus/firebird#95]
when i a am at the fire bird app, and i change the to another the app, the aplication  restart each time don´t save the snapshot , i think add a settting auto save, also, if it's  posiible change the size of screen on tablets when it is portrait and set keyboad have more space

**Comment by Vogtinator** [comment]
This is unfortunately not easily possible on android.
The app takes around 200MiB of RAM (even while running in background) so android kills it pretty much instantly when it's not in the foreground anymore.
In theory the ```onStop()``` Activity callback or the ```QGuiApplication::applicationStateChanged``` signal (in Qt) would work, but creating a snapshot might take too long and either android or the (impatient) user might kill firebird, resulting in a corrupted snapshot.

I'll try to make it a configurable option though, maybe it works better than expected...

**Comment by Vogtinator** [comment]
> I'll try to make it a configurable option though, maybe it works better than expected...

Sadly it does not. Firebird gets shot down just a few seconds after trying to write the snapshot, resulting in a corrupted file every time. For reference, I pushed some code in a separate branch, https://github.com/nspire-emus/firebird/tree/androidsuspend

I enabled suspend on shutdown on android now, but it only works when exiting firebird with the back button though.

So unless someone who knows Android much better than I do (which shouldn't be hard...) comes up with a solution, this is a WONTFIX sadly.
